signing_service/signing_service/signing_service.py

```diff
@@ -267,7 +281,26 @@ def _handle_connection(
                 middleman_message_response = self._prepare_error_response(ErrorCode.UnexpectedMessage, exception)
             # If received frame is correct, validate transaction.
             else:
+                self._update_last_signed_transaction_date(self.last_signed_transaction)
                 golem_message_response = self._get_signed_transaction(middleman_message.payload)
+                if isinstance(golem_message_response, SignedTransaction):
+                    transaction_sum_combined = self.signing_service_daily_transaction_sum_so_far + middleman_message.payload.value
+                    if transaction_sum_combined > MAXIMUM_DAILY_THRESHOLD:
```

Comment:
Please reverse conditions here and below.

Reply:
Done